docs/api/tutorials/sdk/search_client.md

```diff
@@ -0,0 +1,157 @@
+# Search
+
+DataHub’s Python SDK makes it easy to search and discover metadata across your data ecosystem. Whether you're exploring unknown datasets, filtering by environment, or building advanced search tools, this guide walks you through how to do it all programmatically.
+
+## What You Can Do
+
+- Search metadata by keyword or structured filters
+- Filter by environment, platform, type, or custom properties
+- Use AND / OR / NOT logic for advanced queries
+- Perform targeted searches using custom fields like URN
```

Comment:
what exactly does this line mean?

Reply:
this was meant for custom filtering -- I'll update this line to be more straghtforward